es6-persistence/src/main/java/com/netflix/conductor/es6/config/ElasticSearchProperties.java

```diff
@@ -205,4 +209,29 @@ private URL toURL(String url) {
             throw new IllegalArgumentException(url + "can not be converted to java.net.URL");
         }
     }
+
+    public List<URI> getURIs() {
```

Comment:
Since this method is only used in `ElasticSearchV6Configuration`, can this be moved to that class?

Reply:
Moved getURIs() from ElasticSearchProperties.java to ElasticSearchV6Configuration.java